lib/MusicBrainz/Server/Constants.pm

```diff
@@ -426,6 +427,8 @@ Readonly our $MAX_INITIAL_TRACKS => 100;
 Readonly our $MAX_POSTGRES_INT => 2147483647;
 Readonly our $MAX_POSTGRES_BIGINT => 9223372036854775807;
 
+Readonly our $MAX_ONELINE_STRING_LENGTH => 1024;
```

Comment:
Is "oneline" a technical term? Is there a reason why you wouldn't just call it `$MAX_POSTGRES_INDEX_VARCHAR` or something to mirror the `INT` and `BIGINT` constants above?

Reply:
> Is "oneline" a technical term?

It is opposed to multiple-lines strings such as annotations which are of variable unlimited length too in Postgres.

> Is there a reason why you wouldn't just call it `$MAX_POSTGRES_INDEX_VARCHAR` or something to mirror the `INT` and `BIGINT` constants above?

It can also be a `TEXT` in Postgres and this is more our decision than a Postgres limitation really.

Reply:
Maybe worth mentioning as a comment in the source code (outside of the commit message) how the constant was derived? (i.e., the "maximum size allowed for the index type" divided by four to account for multibyte characters).

Reply:
Added comments for both constants. See https://github.com/metabrainz/musicbrainz-server/compare/182f1255fa7f9ebf63853b353812f5e0d26d4033..50f59153b7f5ca57a5cbf4f943b917307aafbc7a